angularjs-portal-home/src/main/webapp/staticFeeds/features.json

```diff
@@ -57,5 +57,23 @@
       "shortDesc" : "If you want to follow NCAA tournaments search for sports in MyUW.",
       "endDate": 1459486800000
     }
+  },
+  {
+    "id": 4,
+    "title": "Wiscard Widget",
+    "description": "MyUW is making your campus life more manageable, one widget at a time. Now you can easily manage your Wiscard account on the go. The new interactive Wiscard widget displays your up-to-date balance and makes it easier for you to add money to your account or report a lost or stolen card. Simply search for \"Wiscard\" and add the interactive widget today!",
+    "img": "img/wiscardWidgets.png",
+    "learnMoreURL": "",
```

Comment:
In which section do I add the link?

Kim


~~~
Kim Miller

Sr. Marketing Specialist
Division of Information Technology
University of Wisconsin⎯Madison
Office: (608) 265-6679 
Cell: (608) 345-7229
www.twitter.com/uwdoit
www.facebook.com/UWDoIT
http://it.wisc.edu/

From:  Tim Levett <notifications@github.com>
Reply-To:  UW-Madison-DoIT/angularjs-portal <reply@reply.github.com>
Date:  Wednesday, April 27, 2016 at 1:42 PM
To:  UW-Madison-DoIT/angularjs-portal <angularjs-portal@noreply.github.com>
Cc:  Kimberly Miller <kim.miller@wisc.edu>, Author <author@noreply.github.com>
Subject:  Re: [UW-Madison-DoIT/angularjs-portal] Adds Wiscard Widget feature announcement (#457)

In angularjs-portal-home/src/main/webapp/staticFeeds/features.json:
> @@ -57,5 +57,23 @@
>        "shortDesc" : "If you want to follow NCAA tournaments search for sports in MyUW.",
>        "endDate": 1459486800000
>      }
> +  },
> +  {
> +    "id": 4,
> +    "title": "Wiscard Widget",
> +    "description": "MyUW is making your campus life more manageable, one widget at a time. Now you can easily manage your Wiscard account on the go. The new interactive Wiscard widget displays your up-to-date balance and makes it easier for you to add money to your account or report a lost or stolen card. Simply search for \"Wiscard\" and add the interactive widget today!",
> +    "img": "img/wiscardWidgets.png",
> +    "learnMoreURL": "",
URL for the search result would be https://my.wisc.edu/web/apps/search/wisccard?q=wisccard

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub